apps/src/blockly/addons/cdoTrashcan.ts

```diff
@@ -28,6 +32,9 @@ export default class CdoTrashcan extends GoogleBlockly.DeleteArea {
      * Current state of lid opening (0.0 = closed, 1.0 = open).
      */
     this.lidOpen_ = 0;
+    this.trashUrl = isDarkTheme(workspace.getTheme())
+      ? this.LIGHT_TRASH_URL
+      : this.TRASH_URL;
```

Comment:
I think we generally label things as assets for dark mode or light mode, so it's a bit confusing that we're using a "light" trash can in dark mode.  I understand why it works in this case, because we happen to uss a lighter asset, but I think it would still be more consistent to label the filename and the URL with "dark" if it's an asset used in dark mode.

Reply:
Thanks for letting me know about these conventions!! I initially had it labeled 'dark' and then felt weird with the lighter color😂 but I'll update now!